### PR TITLE
If cookies disabled, JS SDK fails to load with no notification to calling client

### PR DESCRIPTION
**Pull request by cleio**
Check if localStorage or cookies are disabled on Napster.js initialization.

### CODE REVIEW

**Comment by BreaGaudioso** [comment]
@cleio Can you have Nikola do a quick code review here before we merge it in?

**Comment by NikolaSmrcek** [comment]
@cleio Looks fine to me.

@BreaGaudioso 

Looks fine for me. But im not sure is it okay that we write that there is error in html. Do we want it that way? Or maybe only alert? Or maybe only in console?

Should be merged after streaming-player.

File: example/client.html [hunk 1]
```diff
@@ -40,6 +40,10 @@
       }
 
       $(document).ready(function() {
+        if (typeof Napster === 'undefined') {
+          $('body').html("There has been an error while loading Napster.js. Please check the console for errors.")
```
Comment (NikolaSmrcek) [inline example/client.html]: @BreaGaudioso 
HTML, alert, console?

**Comment by NikolaSmrcek** [comment]
@cleio @BreaGaudioso
Nevermind my mistake. That's in our example client.html

Giving thumbs up for merge. Waiting on steraming-player.

**Pull request merged**